api/v1alpha1/lvmcluster_types.go

```diff
@@ -68,6 +68,10 @@ type ThinPoolConfig struct {
 	// It can be between 64Ki and 1Gi due to the underlying limitations of lvm2.
 	// +optional
 	ChunkSize *resource.Quantity `json:"chunkSize,omitempty"`
+
+	// MetadataSize specifies metadata size for thin pool.
+	// +optional
+	MetadataSize *resource.Quantity `json:"metadataSize,omitempty"`
```

Comment:
This is missing an explanation on what is used when it is nil as well as minimum and maximum sizes